### PR TITLE
Deprecate 1.x balenaOS documentation

### PR DESCRIPTION
**Pull request by vipulgupta2048**
- minor: Deprecate 1.x balenaOS documentation
- Remove os_version dictionary
- Remove 1.x balenaOS tips and workarounds

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

File: pages/reference/OS/overview.md [hunk 1]
```diff
@@ -1,13 +1,202 @@
...
-layout: os-version-specific.html
-
-dynamic:
-  variables: [ $os_version ]
-  ref: $original_ref/$os_version
-  $switch_text: Select a version $os_version
 ---
 
-{{import "overview"}}
+# What is {{ $names.os.lower }}?
```
Comment (klutchell) [inline pages/reference/OS/overview.md:6]: This file hasn't changed much, it was just moved, correct?

**Comment by klutchell** [comment]
lgtm!

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

**Pull request merged**